framework/meson.build

```diff
@@ -1,16 +1,22 @@
 # -*- indent-tabs-mode: t -*-
 # SPDX-License-Identifier: Apache-2.0
 
+sysdeps_dir = 'sysdeps/' + target_machine.system()
+
+if target_machine.system() == 'cygwin'
+  sysdeps_dir = 'sysdeps/windows'
+endif
+
 framework_incdir = include_directories([
-	'sysdeps/' + target_machine.system(),
 	'..',
 	'.',
+	sysdeps_dir,
```

Comment:
This section will conflict now. Please rebase.

Reply:
Rebased